commerce_invoice.info.inc

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * Implements hook_entity_property_info().
+ */
+function commerce_invoice_entity_property_info() {
+  $info = array();
+
+  // Add meta-data about the basic commerce_invoice properties.
+  $properties = &$info['commerce_invoice']['properties'];
+
+  $properties['status'] = array(
```

Comment:
This was already there as `invoice_status` - see `\Drupal\commerce_invoice\Entity\InvoiceMetadataController`